docs/API.md

```diff
@@ -658,8 +658,15 @@ An object with the following properties:
 
 A *test id*. 
 
-If the test has been run with the same domain name within an interval of 10 mins (hard coded), 
-then the new request does not trigger a new test, but returns with the results of the last test
+If a test has been run with the same parameters (as listed below) not more than
```

Comment:
There's an ambiguity here between when a test is requested and when it is started. I think "requested" would be a better word than "run".

Reply:
I have updated. @mattias-p, please re-review.